frontend/server/src/DAO/Problems.php

```diff
@@ -828,20 +828,44 @@ public static function getAdminUser(\OmegaUp\DAO\VO\Problems $problem): ?array {
         ];
     }
 
+    /**
+     * @return list<\OmegaUp\DAO\VO\Problems>
+     */
+    public static function getAllWithTotal(
+        int $page,
+        int $pageSize,
+        int &$total
+    ) {
+        /** @var int $total */
```

Comment:
```suggestion
        /** @var int */
```

en una asignación, no es necesario ponerle el nombre fe la variable.

Reply:
Listo